test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -136,7 +136,7 @@ class ApiUsers::PermissionsControllerTest < ActionController::TestCase
       }
 
       assert_redirected_to api_user_applications_path(api_user)
-      assert_same_elements [application.signin_permission, new_permission], api_user.reload.supported_permissions
+      assert_same_elements [application.signin_permission, new_permission], api_user.supported_permissions
```

Comment:
Is this a change we can make in the non-API tests too (maybe in a separate PR)?

Reply:
I think so - will do this in a separate PR.